docs/topics/spiders.rst

```diff
@@ -377,7 +377,10 @@ Crawling rules
 .. class:: Rule(link_extractor, callback=None, cb_kwargs=None, follow=None, process_links=None, process_request=None)
 
    ``link_extractor`` is a :ref:`Link Extractor <topics-link-extractors>` object which
-   defines how links will be extracted from each crawled page.
+   defines how links will be extracted from each crawled page. Each produced link will
+   be used to generate a :class:`~scrapy.http.Request` object, which will contain the
+   link's text in its ``meta`` attribute.
+   The link text can be accessed from the callback method though ``response.meta['link_text']``
```

Comment:
I wonder if it may make more sense to add this documentation to the description of the ``callback`` parameter below, instead of ``link_extractor``.

Reply:
On second thought, I removed that particular line, it seemed a bit repetitive.